cluster-provision/images/vm-image-builder/README.md

```diff
@@ -40,16 +43,30 @@ then publish it to a registry. In this case a local registry:
 publish-containerdisk.sh example localhost:1234/myimage:mytag
 ```
 
+To build `example` for arm64(aarch64), only need to set the environmental
```

Comment:
```suggestion
To build for arm64(aarch64), you need to set the following environmental
```

Reply:
If we're touching the documentation, I suggest also replacing all instances of "environmental variable" with "environment variable", which AFAIK is the most commonly used spelling.

Reply:
I read it as `enviroment` - tricked my own brain - thanks for catching that. 

You're right I'll update the suggestion.